resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/configuration/CircuitBreakerConfigurationProperties.java

```diff
@@ -211,6 +215,9 @@ public InstanceProperties getBackendProperties(String backend) {
 		@Nullable
 		private Boolean automaticTransitionFromOpenToHalfOpenEnabled;
 
+		@Nullable
+		private Boolean disableStackTraces;
```

Comment:
Can we rename it to `writableStackTraceEnabled` analog to `automaticTransitionFromOpenToHalfOpenEnabled`.